nemo_curator/utils/distributed_utils.py

```diff
@@ -524,16 +536,24 @@ def single_partition_write_with_filename(df, output_file_dir, output_type="jsonl
                         lines=True,
                         force_ascii=False,
                     )
+
             elif output_type == "parquet":
                 output_file_path = output_file_path + ".parquet"
                 out_df.to_parquet(output_file_path)
+
             else:
                 raise ValueError(f"Unknown output type: {output_type}")
 
     return success_ser
 
 
-def write_to_disk(df, output_file_dir, write_to_filename=False, output_type="jsonl"):
+def write_to_disk(
+    df,
+    output_file_dir,
+    write_to_filename=False,
+    keep_filename_column=False,
```

Comment:
Interested in others' opinions here. I think it makes the most sense to default to false here, and automatically drop the "filename" column unless specified otherwise. It saves storage space and the column itself means nothing to the user who can already see the filename after it is written.

I know a lot of our classifier scripts, etc. use the `write_to_disk`, so those will all now automatically default to `keep_filename_column=False`. Don't know if we should consider adding it to `argparse` for more user control on the CLI side, or if folks think we should be good without.

Reply:
I agree, defaulting to `False` is good.

Reply:
Agreed, this behavior makes a lot more sense